readme.md

```diff
@@ -24,6 +24,7 @@ Features and capabilities provided by the plugin include:
 - **Core Theme Support**: enabling AMP compatibility for all core themes, from Twenty Ten all the way through Twenty Twenty.
 - **Compatibility Tool**: when automatic conversion of markup to AMP is not possible, debug AMP validation errors with detailed information including the invalid markup and the specific components responsible on site (e.g theme, plugin, embed); validation errors are shown contextually with their respective blocks in the editor.
 - **CSS Tree Shaking**: automatically remove the majority of unused CSS to bring the total under AMP's 75KB limit; when the total after tree shaking is still over this limit, prioritization is used so that the all-important theme stylesheet important is retained, leaving less important ones to be excluded (e.g. print styles).
+- **Server-side Rendering**: full integration of the [AMP Optimizer library](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) providing automated server-side rendering of AMP pages, drastically improving the [Last Contenful Paint](https://web.dev/lcp/) metric.
```

Comment:
```suggestion
- **Server-side Rendering**: integrates a PHP-port of the [AMP Optimizer](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) providing automated [server-side rendering](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering/) of AMP pages, drastically improving the [Last Contentful Paint](https://web.dev/lcp/) (LCP) metric.
```

Reply:
Note I've also opened a PR to add a mention of the AMP plugin in the AMP docs for SSR: https://github.com/ampproject/amp.dev/pull/4076